rskj-core/src/main/java/co/rsk/net/MessageChannel.java

```diff
@@ -18,13 +18,12 @@
 
 package co.rsk.net;
 
-import co.rsk.net.eth.RskMessage;
 import co.rsk.net.messages.Message;
 
 import java.net.InetAddress;
 
 public interface MessageChannel {
-    void sendMessage(RskMessage rskMessage);
```

Comment:
You did this change and now you're undoing it. I suggest squashing this commit with 6a9f1b5e41774001845bf8c3e63c3ea14fce36a8